docs/configuration/git-providers/github-cloud.md

```diff
@@ -1,22 +1,30 @@
 # GitHub Cloud
 
-## Create a GitHub App
+Follow the instructions below to set up the Codacy integration with GitHub Cloud:
 
-To integrate with GitHub we use a GitHub App.
+1.  Follow the instructions on [creating a GitHub App](create-github-app.md).
 
-Follow the guide to [create the application on GitHub](create-github-app.md).
+2.  Edit the file `values-production.yaml`, set `global.github.enabled: "true"` and define the remaining values with the information obtained when you created the GitHub App:
 
-## `values.yaml` configuration
+    ```yaml
+    github:
+      enabled: true
+      clientId: Iv1.0000000000000000 # Client ID
+      clientSecret: a000000000000000 # Client secret
+      app:
+        name: Codacy # GitHub App name
+        id: 00000 # App ID
+        privateKey: >
+          -----BEGIN RSA PRIVATE KEY-----
+          # Private key (contents of the .pem file)
+          -----END RSA PRIVATE KEY-----
+    ```
```

Comment:
This is not ideal, but I'm trying to provide as much guidance as possible to the users on how to configure their values on the file by using these "dummy values" as templates.